scripts/lokalise.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+const fs = require('fs');
+
+const { LokaliseApi } = require('@lokalise/node-api');
+const admZip = require('adm-zip');
+const request = require('superagent');
+
+const API_KEY = 'd3ca1f966543108a2df870f0f0a85897b9dddaf0';
```

Comment:
Shouldn't we use this as an environment variable rather than hard-coded?

Reply:
JazakAllah Kahiran @abdullahceylan for reviewing. The keys are fixed and will not likely to ever change and also any contributor who wishes to sync his locale files with Lokalise remote locale files will need them so I thought it's okay to include them in the repo. But it's not the best practices for API keys in general I am aware.

Reply:
JazakAllah Khair @osamasayed for explaining. I totally agree with keeping the API key for contributors but maybe you can add it to the `.env` file and use it from there? Just a thought.